bigbluebutton-html5/public/locales/en.json

```diff
@@ -241,6 +241,8 @@
     "app.presentationUploder.dropzoneImagesLabel": "Drag images here to upload",
     "app.presentationUploder.browseFilesLabel": "or browse for files",
     "app.presentationUploder.browseImagesLabel": "or browse/capture for images",
+    "app.presentationUploder.externalUploadTitle": "Add content from Next Cloud",
```

Comment:
Will this function be restricted to "Next Cloud"?
If other external services will be able take advantage of it, this title could be something generic instead of named Next Cloud.

Reply:
it could be changed to something like "Add content from external application"

Reply:
Agreed, or maybe "Add content from 3rd party application". Whatever you like is fine!

Reply:
changed in [a61f0ec](https://github.com/bigbluebutton/bigbluebutton/pull/15251/commits/a61f0ec49b25a8a98773b256328e543e0dbcc51c)